packages/js/api/src/models/orders/shared/types.ts

```diff
@@ -28,10 +28,16 @@ export type OrderDataUpdateParams = 'id' | 'parentId' | 'status' | 'currency' |
 export type OrderTotalUpdateParams = 'total' | 'totalTax';
 
 /**
- * Order address properties
+ * Billing address properties
  */
-export type OrderAddressUpdateParams = 'firstName' | 'lastName' | 'companyName' | 'address1'
-	| 'address2' | 'city' | 'state' | 'postCode' | 'countryCode' | 'email' | 'phone';
+export type BillingOrderAddressUpdateParams = 'firstName' | 'lastName' | 'company' | 'address1'
```

Comment:
This could be defined as `type BillingOrderAddressUpdateParams = ShippingOrderAddressUpdateParams | 'email' | 'phone';`

Reply:
Smart! Done https://github.com/woocommerce/woocommerce/pull/31665/commits/cd2ed1720035678094f01759435d078479c1caef